sql-parser/src/test/java/io/crate/sql/parser/TestStatementBuilder.java

```diff
@@ -197,6 +198,35 @@ public void testKillAll() throws Exception {
         assertTrue(stmt.equals(new KillStatement()));
     }
 
+    @Test
+    public void testDeallocateStmtBuilder() {
+        printStatement("deallocate all");
+        printStatement("deallocate prepare all");
+        printStatement("deallocate 'myStmt'");
+        printStatement("deallocate myStmt");
+        printStatement("deallocate ?");
+        printStatement("deallocate $1");
+    }
+
+    @Test
+    public void testDeallocateStmtWithoutParam() {
```

Comment:
can you also indicate in the test name what's the expected output? eg 'testDeallocateWithoutParamThrowsParsingException'